src/sentry/api/endpoints/organization_details.py

```diff
@@ -248,6 +297,55 @@ def validate(self, attrs):
                 )
         return attrs
 
+    def save_trusted_relays(self, incoming, changed_data, organization):
+        timestamp_now = (
+            datetime.utcnow().isoformat() + "Z"
+        )  # make it utc relative with set timezone
+        option_key = "sentry:trusted-relays"
+        try:
+            # get what we already have
+            existing = OrganizationOption.objects.get(organization=organization, key=option_key)
```

Comment:
_nit_: You could get rid of the following branch if you coerce the above to `()` or `[]`:

```suggestion
            existing = OrganizationOption.objects.get(organization=organization, key=option_key) or ()
```

Reply:
yes, you're right